zebra-chain/src/transparent/tests/prop.rs

```diff
@@ -45,3 +46,25 @@ fn input_coinbase_vecs_only_have_coinbase_input() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn coinbase_height_round_trip() -> Result<()> {
```

Comment:
Test Coverage: Can you please add a byte->height->byte round-trip test?

We need to start with bytes, so we can find decoding bugs that allow multiple encodings for a single height.

Here's a similar test:
https://github.com/ZcashFoundation/zebra/blob/b1acdc61b4791baed5909809e860099ef6782597/zebra-network/src/protocol/external/tests/prop.rs#L46

Reply:
https://github.com/ZcashFoundation/zebra/pull/3129/commits/a40815d236d6bab5053737b0c73f3c55b44a40e1